### PR TITLE
[#141943135]add optional `assessment` section to declaration questions

### PR DESCRIPTION
**Pull request by risicle**
Story https://www.pivotaltracker.com/story/show/141943135

This `assessment` section is intended to be used to denote the "correct" answers for a declaration question and in future will be used to generate a json schema which will perform the automatable component of the assessment.

The `discretionary` flag can be used to mark whether failure of a particular question would simply put the declaration result into a requires-manual-checking state. If not `discretionary` a failure for an answer to match a term in `passIfIn` would simply case a "definite failure". I was thinking that `discretionary` would implicitly default to `false` meaning on most questions it wouldn't be needed.

A question doesn't require a `assessment` section at all if it has no bearing on the assessement, e.g. informational fields like email addresses.

@TheDoubleK and I decided that the term `assessment` was clearest/least confusing/ambiguous, but we could discuss this further.

*This PR is intended to include the filling out of these fields for all of G9, but am putting the schema change up early to give someone a chance to raise objections before I've gone through the whole repo annotating the correct answers in this format.*

### CODE REVIEW

**Comment by TheDoubleK** [comment]
Don't give them too much of a chance... :+1:

File: frameworks/g-cloud-9/questions/declaration/servicesHaveOrSupport.yml [hunk 1]
```diff
@@ -8,3 +8,6 @@ question: >
     - the delivery of cloud services. Your service must enable the delivery of cloud services. Your service complies with the definition of cloud support in the <a href="/suppliers/frameworks/g-cloud-9" target="_blank" rel="noopener noreferrer">framework agreement (link opens in a new tab)</a> (Lot 3 - cloud support.)
 
 type: boolean
+assessment:
+  passIfIn:
+    - True
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/declaration/servicesHaveOrSupport.yml]: Missing line terminator here.

File: frameworks/g-cloud-9/questions/declaration/taxEvasion.yml [hunk 2]
```diff
@@ -4,3 +4,7 @@ type: boolean
 hint: >
   ‘Legally found’ means any judicial or administrative decision, which has final or binding effect. Check that your organisation complies with 
   <a href="http://www.legislation.gov.uk/uksi/2015/102/regulation/57/made" target="_blank" rel="noopener noreferrer">Regulation 57(3) of the Public Contracts Regulations 2015 (link opens in a new tab)</a>.
+assessment:
+  passIfIn:
+    - False
+  discretionary: True
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/declaration/taxEvasion.yml]: Missing line terminator here.

**Comment by risicle** [comment]
Yup will fix newlines

**Comment by TheDoubleK** [comment]
Good once Travis is happy.

**Comment by samuelhwilliams** [comment]
Will need a version bump.

**Comment by risicle** [comment]
*You'll* need a version bump

**Comment by risicle** [comment]
I'm bloody doing it!

**Pull request merged**